resources/dictionary.txt

```diff
@@ -1,3 +1,4 @@
+Acknowledgement
```

Comment:
Acknowledgment?

Reply:
just american vs british - either way is fine, just pointing it out :)

Reply:
This one always trips me up, thanks!!